### ISSUE TITLE
Set Course Search input to type search

### ISSUE THREAD

**Issue by atcupps** [atcupps/Jupiterp#83]
Course Search's input box currently has `type='text'`; it should be changed to `type='search'`. This may have some screen reader benefits (not sure), and will allow a small X button to clear the input which is more convenient than having to manually backspace all the letters each time to add a course. As part of merging this change, the clear button should be styled appropriately to fit with the page.

**Comment by atcupps** [comment]
Actually going to close this issue since I don't think it would be as beneficial as I initially thought. Seems to only really affect desktop chrome?
